brainscore_language/__init__.py

```diff
@@ -69,4 +69,21 @@ def score(model_identifier: str, benchmark_identifier: str) -> Score:
     score: Score = benchmark(model)
     score.attrs['model_identifier'] = model_identifier
     score.attrs['benchmark_identifier'] = benchmark_identifier
+    CondaScore.save_score(score)
+    
     return score
+
+
+def score(model_identifier: str, benchmark_identifier: str, install_dependencies='yes') -> Score:
+    """ 
+    install_dependencies='yes' installs dependencies directly into runtime environment
+    install_dependencies='newenv' runs score() in an auto-generated conda environment
+    install_dependencies='no' leaves dependency installation to the user
+    """
+    if install_dependencies == 'newenv':
+        result = CondaScore(model_identifier, benchmark_identifier).score
+    else:
+        os.environ['BSL_DEPENDENCY_INSTALL'] = os.getenv('BSL_DEPENDENCY_INSTALL', install_dependencies)
```

Comment:
> Setting os.environ from inside the program is very confusing imo.

Do you mean as opposed to having a user set it explicitly before running, or just setting it in `__main__` instead of `__init__`? I figured the user-friendly option is using the `--install_dependencies` flag via CLI, but I realized that with all the plugins calling `load_plugin`, it's much nicer to have an environment variable set rather than throwing an `install_dependencies` param around everywhere. I agree that it makes more sense to set this in `__main__`, so it's now 
```
def score(model_identifier: str, benchmark_identifier: str, install_dependencies=None):

    os.environ['BSL_INSTALL_DEPENDENCIES'] = install_dependencies or 'yes'
```
(While I feel like it's typically nicer to set env vars with `os.getenv` so you can defer to user input, I realized we need to be able to update this on runs started with `newenv`.)